lib/react_on_rails/configuration.rb

```diff
@@ -11,17 +11,21 @@ def self.configuration
       logging_on_server: true,
       generator_function: false,
       trace: Rails.env.development?,
+      reload_server_js_every_request: Rails.env.development?,
```

Comment:
Question - I change this on line 14.

what about here:
```
      if development_mode.nil?
        self.reload_server_js_every_request = Rails.env.development?
      else
        self.reload_server_js_every_request = development_mode
      end
```

Do we want to rename out internal variable development_mode also?